agilent_infiniium_scope.py

```diff
@@ -366,18 +371,27 @@ def get_trace(self, c, channel, start=1, stop=10000):
         #Transfer waveform preamble
         preamble = yield dev.query('WAV:PRE?')
         #Transfer waveform data
-        binary = yield dev.query('WAV:DATA?')
+        p = dev._packet(). \
+            write('WAV:DATA?'). \
+            read_raw()
+
+        result = yield p.send()
+        binary = result['read_raw']
+        binary = binary[:-1]
```

Comment:
Probably not the correct way to handle this, but I believe that there is a single termination character at the end of the byte string so I drop it manually.